Snakefile

```diff
@@ -129,14 +129,15 @@ rule transform_genbank_data:
     output:
         fasta = "data/genbank/sequences.fasta",
         metadata = "data/genbank/metadata_transformed.tsv",
+        flagged_annotations = temp("data/genbank/flagged-annotations"),
         duplicate_biosample = "data/genbank/duplicate_biosample.txt"
     shell:
         """
         ./bin/transform-genbank {input.ndjson} \
             --biosample {input.biosample} \
             --duplicate-biosample {output.duplicate_biosample} \
             --output-metadata {output.metadata} \
-            --output-fasta {output.fasta}
+            --output-fasta {output.fasta} > {output.flagged_annotations}
```

Comment:
Should we redirect stderr too?

Or even `tee` it so it's visible in Slack and in the logs? That might be most useful. We want to know if there are errs.

Reply:
@corneliusroemer The goal of the PR is to add that 1 statement that was forgotten when copied from gisaid script, ant to allow engineers to read logs.

I'd avoid messing with stderr here, but we may consider it if it is needed in a separate PR. In that case we'd probably better tp upload error logs for all stages and all databases, not just this one thing. Transform stage is actually somewhat stable and unlikely to emit errors.